src/Trips.js

```diff
@@ -0,0 +1,36 @@
+class Trips {
+  constructor(userTrips) {
+    this.allTrips = userTrips;
+    this.pending = [];
+    this.approved = [];
+    this.past = [];
+    this.future = [];
+  }
+
+  findTripById(id) {
+    return this.allTrips.find((trip) => {
+      return trip.id === id;
+    })
+  }
+
+  sortTripsByStatus() {
+    return this.allTrips.forEach((trip) => {
+      this[trip.status].push(trip);
```

Comment:
Just curious if you had to bring in your Trip class into this file in order to get this test to pass with trip.status? Very creative!